airbyte-cdk/python/unit_tests/sources/test_source.py

```diff
@@ -390,6 +390,7 @@ def test_internal_config_limit(abstract_source, catalog):
     logger_mock.level = logging.DEBUG
     del catalog.streams[1]
     STREAM_LIMIT = 2
+    SLICE_DEBUG_LOG_COUNT = 1
```

Comment:
These tests should have been fixed in https://github.com/airbytehq/airbyte/pull/21525 but I'm doing this here for simplicity's sake